package.json

```diff
@@ -4,9 +4,10 @@
   "description": "Test and sample webapp for the Createjs Accessibility Module",
   "main": "src/index.js",
   "scripts": {
-    "build": "rm -rf dist && NODE_ENV=production webpack",
+    "build": "rm -rf dist && npm run lint && NODE_ENV=production webpack",
```

Comment:
From checking with Josh, the way the linter should be integrated with the build is by adding a line above the existing build script https://github.com/CurriculumAssociates/createjs-accessibility/blob/master/.travis.yml#L5 to kick off the linter.